packages/fxa-graphql-api/src/config.ts

```diff
@@ -114,6 +114,32 @@ const conf = convict({
       },
     },
   },
+  metrics: {
```

Comment:
The fxa-auth-server config under statsd has a sample rate option as well, because its high-traffic. I don't think we've ever needed to change it though.

Reply:
I'll add it in just in case.

Reply:
Just double checking, is there a difference between 'METRIC_*' and 'STATSD_*' envs? I am noticing that the auth server uses STATSD prefix, while event broker uses the METRIC prefix convention.